css/quotes.css

```diff
@@ -1,4 +1,180 @@
-/* Quote page specific styles */
+* {
+    box-sizing: border-box;
+}
 
+body {
+    margin: 0;
+}
 
-/* We should use a mobile first approach having out breack point at 768px */
\ No newline at end of file
+.form {
+    margin-left: 5%;
+    margin-right: 5%;
+    max-width: 90%;
+}
+
+.form_title {
+    font-size: 22px;
+    font-weight: bold;
+    font-family: 'Roboto', sans-serif;
+    text-align: justify;
+    margin-top: 50px;
+    margin-bottom: 10px;
+    color: rgba(0, 0, 0, 0.7)
+}
+
+.form_subtitles {
+    font-weight: bold;
+    font-family: 'Roboto', sans-serif;
+    text-align: justify;
+    margin-bottom: 18px;
+    color: rgba(0, 0, 0, 0.7)
+}
+
+hr {
```

Comment:
Please use a class instead of a tag directly. This could affect other components.